docs/faq.md

```diff
@@ -171,7 +171,23 @@ mappings for your backend services. You can work around this problem using the
 `cluster_tag` property, as described in
 [emissary-ingress/emissary#3112](https://github.com/emissary-ingress/emissary/issues/3112).
 
+### HTTP 464 error with AWS
+
+If you're using the [Application Load Balancer support for gRPC][alp-aws-grpc], 
+you will likely see an HTTP error response with code 464 for a Connect `GET` 
+request, or for a web browser making a CORS preflight `OPTIONS` request.
```

Comment:
Would it be useful to link out to an explanation of when Connect does these actions? (Such as how to opt in to GET support)

Reply:
I considered linking to:
- the blog post
- protocol definition for Unary-Get-Request
- Go / Get Requests and Caching
- Node.js / Get Requests and Caching

None of them seems like a good fit to me.

Reply:
Alright, probably okay to leave it out then